zabbix/repo.sls

```diff
@@ -24,13 +24,14 @@
     - name: apt-transport-https
 {{ id_prefix }}_repo:
   pkgrepo.managed:
-    - name: deb https://repo.zabbix.com/zabbix/{{ zabbix.version_repo }}/{{ salt['grains.get']('os')|lower }} {{ salt['grains.get']('oscodename') }} main
+    - name: deb [ arch={{ salt['grains.get']('osarch')|lower }} {{ zabbix.repo_signedby }} ] https://repo.zabbix.com/zabbix/{{ zabbix.version_repo }}/{{ salt['grains.get']('os')|lower }} {{ salt['grains.get']('oscodename') }} main
     - file: /etc/apt/sources.list.d/zabbix.list
     - key_url: https://repo.zabbix.com/zabbix-official-repo.key
     - clean_file: True
+    - aptkey: {{ zabbix.repo_aptkey }}
```

Comment:
If I'm not mistaken `aptkey` added in Salt 3005, have you tested your changes in Salt 3004?

Reply:
I have tested it works find on 3004.2. It just gets ignored.